File: function.php
// function.php
function myFunction($param) {
    $x = $param;
    $x += 10;
    if ($x > 15) {
        $x -= 5;
    } else {
        $x += 5;
    }
    for ($i = 0; $i < 3; $i++) {
        $x += $i;
    }
    return $x;
}

echo "Enter an integer: ";
$input = trim(fgets(STDIN));
$param = intval($input);

if ($param == 0 && $input !== "0") {
    echo "Invalid input\n";
} else {
    echo myFunction($param) . "\n";
}

File: function.pl
# function.pl
sub my_function {
    my ($param) = @_;
    my $x = $param;
    $x += 10;
    if ($x > 15) {
        $x -= 5;
    } else {
        $x += 5;
    }
    for my $i (0..2) {
        $x += $i;
    }
    return $x;
}

print "Enter an integer: ";
my $input = <STDIN>;
chomp($input);
my $param = int($input);

if ($param == 0 && $input ne "0") {
    print "Invalid input\n";
} else {
    print my_function($param) . "\n";
}
